lib/driver.js

```diff
@@ -413,14 +413,9 @@ class AndroidUiautomator2Driver extends BaseDriver {
     return `${WEBVIEW_BASE}0`;
   }
 
-  async setCompressedLayoutHierarchy (compress) {
-    await this.uiautomator2.jwproxy.command('/appium/device/compressedLayoutHierarchy', 'POST', {compressLayout: compress});
-  }
-
   async onSettingsUpdate (key, value) {
-    if (key === "ignoreUnimportantViews") {
-      await this.setCompressedLayoutHierarchy(value);
-    }
+    let settings = {[key]: value};
+    await this.uiautomator2.jwproxy.command('/appium/settings', 'POST', {settings: settings});
```

Comment:
`{settings: settings}` can just be `{settings}` (see https://github.com/getify/You-Dont-Know-JS/blob/master/es6%20%26%20beyond/ch2.md#concise-properties)